libraries/AP_Proximity/AP_Proximity_Backend.h

```diff
@@ -69,39 +68,19 @@ class AP_Proximity_Backend
 
     // correct an angle (in degrees) based on the orientation and yaw correction parameters
     float correct_angle_for_orientation(float angle_degrees) const;
-
-    // find which sector a given angle falls into
-    uint8_t convert_angle_to_sector(float angle_degrees) const;
-
-    // initialise the boundary and sector_edge_vector array used for object avoidance
-    //   should be called if the sector_middle_deg or _setor_width_deg arrays are changed
-    void init_boundary();
-
-    // update boundary points used for object avoidance based on a single sector's distance changing
-    //   the boundary points lie on the line between sectors meaning two boundary points may be updated based on a single sector's distance changing
-    //   the boundary point is set to the shortest distance found in the two adjacent sectors, this is a conservative boundary around the vehicle
-    void update_boundary_for_sector(const uint8_t sector, const bool push_to_OA_DB);
-
+    
     // check if a reading should be ignored because it falls into an ignore area
     // angles should be in degrees and in the range of 0 to 360
     bool ignore_reading(uint16_t angle_deg) const;
-
+    
     // database helpers.  all angles are in degrees
-    bool database_prepare_for_push(Vector3f &current_pos, Matrix3f &body_to_ned);
-    void database_push(float angle, float distance);
-    void database_push(float angle, float distance, uint32_t timestamp_ms, const Vector3f &current_pos, const Matrix3f &body_to_ned);
+    static bool database_prepare_for_push(Vector3f &current_pos, Matrix3f &body_to_ned);
+    static void database_push(float angle, float distance);
```

Comment:
we should add a comment to clarify that the angle is a yaw angle in "body frame"